src/service/Database.ts

```diff
@@ -0,0 +1,68 @@
+import AWS from 'aws-sdk';
+import { DatabaseOperationStatus } from '../model/database';
+import { user } from '../model/user';
+
+require('dotenv').config();
+
+const creds = new AWS.Credentials(process.env.AWSACCESSKEYID, process.env.AWSSECRETKEY);
+AWS.config.update({
+  region: 'eu-west-3',
+  credentials: creds,
+});
+
+const documentClient = new AWS.DynamoDB.DocumentClient();
+
+export const createUser = async (userToCreate:user):Promise<DatabaseOperationStatus> => {
+  const params = {
+    TableName: 'BlockChainUsers',
+    Item: {
+      PublicKey: userToCreate.PublicKey,
+      PrivateKey: userToCreate.PrivateKey,
+    },
+    ReturnValues: 'ALL_OLD',
+    ReturnConsumedCapacity: 'TOTAL',
+  };
+  try {
+    await documentClient.put(params).promise();
+    return DatabaseOperationStatus.success;
+  } catch (exceptionCaught) {
+    console.error(exceptionCaught);
+    return DatabaseOperationStatus.failure;
+  }
+};
+
+export const scanTable = async ():Promise<any[]> => {
+  const params : AWS.DynamoDB.Types.ScanInput = {
+    TableName: 'BlockChainUsers',
+  };
+  const scanResults:any[] = [];
+  const items = await documentClient.scan(params).promise();
+  do {
+    items.Items.forEach((item:any) => scanResults.push(item));
+    params.ExclusiveStartKey = items.LastEvaluatedKey;
+  } while (typeof items.LastEvaluatedKey !== 'undefined');
+  return scanResults;
+};
+
+const onScan = (users:user[]) => (
```

Comment:
type the function

Reply:
function of function , void return defined just after